migrations/00004_provider-request-generic.sql

```diff
@@ -0,0 +1,13 @@
+-- start a transaction
+BEGIN;
+
+ALTER TABLE
+  auth.provider_requests
+DROP COLUMN redirect_url;
+
+ALTER TABLE
+  auth.provider_requests
+ADD
+  COLUMN metadata JSONB;
```

Comment:
Either we use a general `metadata JSONB` column or create individual columns for:

- redirectTo
- displayName
- locale
- defaultRole
- allowedRoles
- metadata

Reply:
It's also safe to do this change later if we want to.

Reply:
I would rather name the column options than metadata as it captures all registration options, including options.metadata, options.redirectTo etc